.github/settings.yml

```diff
@@ -0,0 +1,24 @@
+# Collaborators: give specific users access to this repository.
+# See https://docs.github.com/en/rest/reference/repos#add-a-repository-collaborator for available options
```

Comment:
This link is out dated and not very useful, we probably want https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permission-levels-for-repositories-owned-by-an-organization instead.

Reply:
I copied this link from https://github.com/tinkerbell/tink/pull/611, which one should I use?